jetty-websocket/websocket-jetty-server/src/main/config/modules/websocket-jetty.mod

```diff
@@ -0,0 +1,21 @@
+DO NOT EDIT - See: https://www.eclipse.org/jetty/documentation/current/startup-modules.html
+
+[description]
+Enable jetty websocket for deployed web applications.
```

Comment:
probably should be clear this is the jetty API and not just the jetty websocket implementation.